importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -4940,15 +4940,12 @@ void MusicXMLParserPass2::harmony(const QString& partId, Measure* measure, const
                         if (_e.name() == "root-step") {
                               // attributes: print-style
                               step = _e.readElementText();
-                              /* TODO: check if this is required
-                              if (ee.hasAttribute("text")) {
-                                    QString rtext = ee.attribute("text");
-                                    if (rtext == "") {
+                              if (_e.attributes().hasAttribute("text")) {
```

Comment:
I suppose this means if the kind tag appears *before* the root-step tag, and there is no "none" on the root-step, we end up setting the root tpc after all.  That's probably a reasonable interpretation of a somewhat ambiguous spec anyhow.

Reply:
Thinking more about it, I would have preferred to only rely on `kind=="none"` to signal an invalid root, since that much is clear. Happy to play it however you see fit though.

Reply:
Anything more to do here on my side?

Reply:
Kind before root-step would not be a valid MusicXML file, the schema does not allow that.
The root-step element is a child of the root element, which must be before the kind element.